README.md

```diff
@@ -16,16 +16,29 @@ You can install via [Sublime Package Manager](https://sublime.wbond.net/) where
 
 ###### OSX
 Toggle Comments Folding: <kbd>Command</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>
+Fold Current Comments: <kbd>Command</kbd> + <kbd>Alt</kbd> + <kbd>C</kbd>
+Unfold Current Comments: <kbd>Command</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>
 
 ###### Windows/Linux
 Toggle Comments Folding: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>
+Fold Current Comments: <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>C</kbd>
+Unfold Current Comments: <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>
 
 
 #### Commands
 
 * `Toggle Folding Comments`
 * `Fold Comments`
 * `Unfold Comments`
+* `Fold Current Comments`
+* `Unfold Current Comments`
+
+
+#### Settings
+
+* `"fold_single_line_comments": true`  # Fold single line comments.
+* `"show_first_line": false`  # Show the first line of multiline comments.
+* `"autofold": false`  # Automatically fold comments when a file is loaded.
```

Comment:
Autofolding should happen when a file is first loaded only. Not every time the tab gains focus